tests_integration/conftest.py

```diff
@@ -19,6 +19,7 @@
 from typer import Typer
 from typer.testing import CliRunner
 
+from tests.conftest import clean_logging_handlers_fixture
 from tests.testing_utils.fixtures import snowflake_home, temp_dir
```

Comment:
I think `temp_dir` is not needed anymore (I had to import it as `snowflake_home` was dependent on it)

Reply:
`temp_dir` is used in `__all__ = ["snowflake_home", "temp_dir",`